ReleaseNotes.rst

```diff
@@ -7,6 +7,32 @@
 
    OldReleaseNotes
 
+Release 30.0
+------------
+Changes
+^^^^^^^
+* Upgrade to Firely .NET SDK 4.3.0.
+* Added Project menu.
+* Redesigned Option menu: moved settings to Settings dialog.
+* Added option **Do this for all current items** for some dialogs:
+
+  - Save file changes: **Save** or **Discard**
+  - Update resource FHIR version: **Yes** or **No**
+* R4B officially supported by Forge
+* **[R4]** Added R4B Upgrade Analysis Wizard in Project menu
+* **[R4B]** Added R4B Upgrade Wizard in Project menu
+
+Bug fixes
+^^^^^^^^^
+* The resource resolver in Forge is not updated after saving modifications in a resource. 
```

Comment:
Let's write all of these in past tense :) 'was not updated', 'was noticeable', ' would be reported again'. Else it could read like known issues list.